CHANGELOG.md

```diff
@@ -6,6 +6,8 @@ The format of this document is based on [Keep a Changelog](https://keepachangelo
 
 ## [Unreleased]
```

Comment:
Maybe this has to be removed ?

Reply:
No, it needs to be there so that following changes that are unreleased will go there.